lib/config/providers/chat_input_provider.dart

```diff
@@ -0,0 +1,116 @@
+import 'dart:async';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:logging/logging.dart';
+import 'package:whitenoise/domain/services/draft_message_service.dart';
+import 'package:whitenoise/domain/services/image_picker_service.dart';
+import 'package:whitenoise/ui/chat/states/chat_input_state.dart';
+
+class ChatInputNotifier extends StateNotifier<ChatInputState> {
```

Comment:
Let's always prefer Notifiers/NotifierProviders (2.0+) over StateNotifiers/StateNotifierProviders (1.x).

Reply:
After rebase now the sha changed... https://github.com/parres-hq/whitenoise_flutter/pull/721/commits/f089764bd5ace78e7252cac05d941cbdd3539271